WordPress/Classes/ViewRelated/Reader/ReaderMenuViewController.swift

```diff
@@ -176,14 +176,42 @@ import WordPressShared
     ///
     func followTagNamed(tagName: String) {
         let service = ReaderTopicService(managedObjectContext: ContextManager.sharedInstance().mainContext)
-        service.followTagNamed(tagName, withSuccess: nil) { (error) in
-            DDLogSwift.logError("Could not follow tag named \(tagName) : \(error)")
 
-            let title = NSLocalizedString("Could not Follow Tag", comment: "Title of a prompt informing the user there was a probem unsubscribing from a tag in the reader.")
-            let message = error.localizedDescription
-            let alert = UIAlertController(title: title, message: message, preferredStyle: .Alert)
-            alert.addCancelActionWithTitle(NSLocalizedString("OK", comment: "Button title. An acknowledgement of the message displayed in a prompt."))
-            alert.presentFromRootViewController()
+        service.followTagNamed(tagName, withSuccess: { [weak self] in
+            // A successful follow makes the new tag the currentTopic.
+            if let tag = service.currentTopic as? ReaderTagTopic {
+                self?.scrollToTag(tag)
+            }
+
+            }, failure: { (error) in
+                DDLogSwift.logError("Could not follow tag named \(tagName) : \(error)")
+
+                let title = NSLocalizedString("Could not Follow Tag", comment: "Title of a prompt informing the user there was a probem unsubscribing from a tag in the reader.")
+                let message = error.localizedDescription
+                let alert = UIAlertController(title: title, message: message, preferredStyle: .Alert)
+                alert.addCancelActionWithTitle(NSLocalizedString("OK", comment: "Button title. An acknowledgement of the message displayed in a prompt."))
+                alert.presentFromRootViewController()
+        })
+    }
+
+
+    /// Scrolls the tableView so the specified tag is in view.
+    ///
+    /// - Paramters:
+    ///     - tag: The tag to scroll into view.
+    ///
+    func scrollToTag(tag: ReaderTagTopic) {
+        guard let indexPath = viewModel.indexPathOfTag(tag) else {
+            return
+        }
+        tableView.selectRowAtIndexPath(indexPath, animated: true, scrollPosition: .Middle)
```

Comment:
This makes me nervous as we're utilizing a method that just _happens_ to create an animated effect that we want, but is intended for selection. It might be best to go ahead and animate the cell's `contentView` background color itself, as a bit more of an intentional animation effect.